src/sidebar/components/group-list.js

```diff
@@ -78,7 +85,16 @@ function GroupListController($window, analytics, groups, settings, serviceUrl) {
   this.isThirdPartyService = isThirdPartyService(settings);
 
   this.showGroupsMenu = () => {
-    return !(this.isThirdPartyService && this.groups.all().length <= 1);
+    if (features.flagEnabled('community_groups')) {
+      // Only show the drop down menu if there is more than one group.
+      return this.groups.all().length > 1;
+    } else {
+      return !(this.isThirdPartyService && this.groups.all().length <= 1);
+    }
+  };
+
+  this.isFeatureFlagEnabled = flag => {
```

Comment:
As we chatted about in person, might be helpful to drop a comment in here to explain that this wrapping is needed for scope (the template needs access to feature-flag info).